src/oracle/oracleDataController.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Route, Tags } from "tsoa";
+import { getLatestSubmission, getLatestSubmissionForEachOracle } from "./oracleDataService";
+import { getPolkaBtc } from "../common/polkaBtc";
+import { OracleStatus } from "./oracleModel";
+
+interface CachedOracleData {
+    maxDelay: number;
+    feed: string;
+    namesMap: Map<string, string>
+}
+
+@Tags("stats")
+@Route("oracle")
+export class OracleDataController extends Controller {
+    cachedOracleData: Promise<CachedOracleData> = (async () => {
+        const polkabtc = await getPolkaBtc();
+        const maxDelay = await polkabtc.oracle.getMaxDelay();
```

Comment:
you can wrap this in a `await Promise.all([])` to execute those calls in parallel.